terraso_backend/apps/story_map/collaboration_roles.py

```diff
@@ -0,0 +1,16 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+ROLE_COLLABORATOR = "collaborator"
```

Comment:
question: is this the same as an editor role?

Reply:
@paulschreiber yes, it changed from contributor to collaborator and the last change was to editor